src/sections/section5/Section5.css

```diff
@@ -0,0 +1,86 @@
+.codeutsava__section5 {
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  font-family: "Poppins";
+  color: white;
+}
+
+.codeutsava__section5-body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.codeutsava__section5-title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 48px;
+  line-height: 82px;
+  text-align: center;
+  letter-spacing: 1px;
+  margin-bottom: 2rem;
+  margin-top: 2rem;
+  color: white;
+}
+
+.codeutsava__section5-content {
+  font-style: normal;
+  font-size: 24px;
+  width: 85%;
+  line-height: 36px;
+  text-align: center;
+  letter-spacing: -1px;
+}
+
+.codeutsava__section5-sponsers {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 4rem 0;
+  gap: 4rem;
+}
+
+.codeutsava__section5-sponser-title {
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+}
+
+.codeutsava__section5-sponser-platnium {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #e5e4e2;
+  padding: 2rem;
+  box-shadow: 0px 20px 50px rgba(18, 17, 39, 0.08);
+  border-radius: 20px;
+  gap: 3rem;
+}
+
+.codeutsava__section5-sponser-gold {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #FFD700;
+  padding: 2rem;
+  box-shadow: 0px 20px 50px rgba(18, 17, 39, 0.08);
+  border-radius: 20px;
+  gap: 3rem;
+}
+
+.codeutsava__section5-sponser-silver {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
+  border: 1px solid #C0C0C0;
+  padding: 2rem;
+  box-shadow: 0px 20px 50px rgba(18, 17, 39, 0.08);
+  border-radius: 20px;
+  gap: 3rem;
+}
```

Comment:
Make a base class rather than repeating same styles in different classes. Also, the sponsors block width should be fixed based upon the screen size.

Reply:
Added a base class with repeated designs.